### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by ktd1008** [lehoangnb/mitm_ecovacs_app_bypass#1]
Bác ơi, có cách nào tạo server local để chạy đc robot ecovacs không bác?

**Comment by lehoangnb** [comment]
> Bác ơi, có cách nào tạo server local để chạy đc robot ecovacs không bác?

B dùng bumper là đc [https://github.com/bmartin5692/bumper](url), mỗi tội cái này mình test thì bị lỗi phần tự động dọn dẹp theo lịch

**Comment by ktd1008** [comment]
> > Bác ơi, có cách nào tạo server local để chạy đc robot ecovacs không bác?
> 
> 
> 
> B dùng bumper là đc [https://github.com/bmartin5692/bumper](url), mỗi tội cái này mình test thì bị lỗi phần tự động dọn dẹp theo lịch

Em dùng t9 aivi với dùng pi3b+ chạy bumper nhưng lúc bật tắt lại thiết bị. Nó lại ko thấy bác ạ. Bác dùng docker hay python?

**Comment by lehoangnb** [comment]
> > > Bác ơi, có cách nào tạo server local để chạy đc robot ecovacs không bác?
> > 
> > 
> > B dùng bumper là đc [https://github.com/bmartin5692/bumper](url), mỗi tội cái này mình test thì bị lỗi phần tự động dọn dẹp theo lịch
> 
> Em dùng t9 aivi với dùng pi3b+ chạy bumper nhưng lúc bật tắt lại thiết bị. Nó lại ko thấy bác ạ. Bác dùng docker hay python?

b sửa DNS chưa? Mình dùng python chạy trên lxc, mà chạy kiểu này cũng lằng nhằng, b bypass cái app bằng proxy rồi dùng server của ecovacs còn đơn giản hơn

**Comment by ktd1008** [comment]
> > > > Bác ơi, có cách nào tạo server local để chạy đc robot ecovacs không bác?
> 
> > > 
> 
> > > 
> 
> > > B dùng bumper là đc [https://github.com/bmartin5692/bumper](url), mỗi tội cái này mình test thì bị lỗi phần tự động dọn dẹp theo lịch
> 
> > 
> 
> > Em dùng t9 aivi với dùng pi3b+ chạy bumper nhưng lúc bật tắt lại thiết bị. Nó lại ko thấy bác ạ. Bác dùng docker hay python?
> 
> 
> 
> b sửa DNS chưa? Mình dùng python chạy trên lxc, mà chạy kiểu này cũng lằng nhằng, b bypass cái app bằng proxy rồi dùng server của ecovacs còn đơn giản hơn

Em có sửa như này 
![image](https://github.com/lehoangnb/mitm_ecovacs_app_bypass/assets/121656368/75f67c29-9c59-44e2-a5c9-6bfae4219fbf)
192.168.1.132 là địa chỉ pi em đang chạy pihole.
Sau đó em chạy tiếp lệnh
pipenv run python -m bumper --listen 192.168.1.132 --announce 192.168.1.132
Không biết em có làm sai chỗ nào ko mà lúc reset lại robot. Nó vẫn ko hiện trong webui. Mong bác chỉ giáo giúp em 😃

**Comment by ktd1008** [comment]
Em muốn chạy bumper rồi sau đó add lên hass bác ạ

**Comment by lehoangnb** [comment]
> Em muốn chạy bumper rồi sau đó add lên hass bác ạ

b thêm --debug  rồi xem bumper nó báo log ra sao lúc reset robot, dns b set như kia thì ok rồi, mình ko dùng hass nên ko rõ, trc giờ toàn dùng app ecovacs thôi

**Comment by lehoangnb** [comment]
Mình có xem qua component ecovacs của hass thì ko dùng đc vs bumper đâu b, phải sửa lại thư viện sucks hoặc b dùng cái này xem đc ko nhé https://github.com/bittles/ha_ecovacs_bumper (nhớ config verify_ssl: false)

**Comment by ktd1008** [comment]
> > Em muốn chạy bumper rồi sau đó add lên hass bác ạ
> 
> 
> 
> b thêm --debug  rồi xem bumper nó báo log ra sao lúc reset robot, dns b set như kia thì ok rồi, mình ko dùng hass nên ko rõ, trc giờ toàn dùng app ecovacs thôi

Vâng. Để tối em đi làm về em chạy lệnh debug xem sao. Còn vấn đề hass. Em nghĩ nó chưa kết nối đc với bumper nên khả năng ko add đc. Qua em cũng ha_ecovacs kia rồi 😃
Còn git của bác. Chạy đc trên raspbian chứ ạ. Em thì ko phải dân chuyên. Chỉ là thích vọv vạch thôi ạ

**Comment by lehoangnb** [comment]
> > > Em muốn chạy bumper rồi sau đó add lên hass bác ạ
> > 
> > 
> > b thêm --debug  rồi xem bumper nó báo log ra sao lúc reset robot, dns b set như kia thì ok rồi, mình ko dùng hass nên ko rõ, trc giờ toàn dùng app ecovacs thôi
> 
> Vâng. Để tối em đi làm về em chạy lệnh debug xem sao. Còn vấn đề hass. Em nghĩ nó chưa kết nối đc với bumper nên khả năng ko add đc. Qua em cũng ha_ecovacs kia rồi 😃 Còn git của bác. Chạy đc trên raspbian chứ ạ. Em thì ko phải dân chuyên. Chỉ là thích vọv vạch thôi ạ

b tắt verify_ssl chưa, phải tắt mới đc, git của mình thì mới test trên linux x86_64 (debian), còn arm thì chưa

**Comment by ktd1008** [comment]
> > > > Em muốn chạy bumper rồi sau đó add lên hass bác ạ
> 
> > > 
> 
> > > 
> 
> > > b thêm --debug  rồi xem bumper nó báo log ra sao lúc reset robot, dns b set như kia thì ok rồi, mình ko dùng hass nên ko rõ, trc giờ toàn dùng app ecovacs thôi
> 
> > 
> 
> > Vâng. Để tối em đi làm về em chạy lệnh debug xem sao. Còn vấn đề hass. Em nghĩ nó chưa kết nối đc với bumper nên khả năng ko add đc. Qua em cũng ha_ecovacs kia rồi 😃 Còn git của bác. Chạy đc trên raspbian chứ ạ. Em thì ko phải dân chuyên. Chỉ là thích vọv vạch thôi ạ
> 
> 
> 
> b tắt verify_ssl chưa, phải tắt mới đc, git của mình thì mới test trên linux x86_64 (debian), còn arm thì chưa

Tắt ở đâu bác. Em có thêm dòng verify_ssl trong configuration.yaml của hass rồi bác. 
Git của bác, chạy rồi có cần config ở đt ko ạ

**Comment by lehoangnb** [comment]
> > > > > Em muốn chạy bumper rồi sau đó add lên hass bác ạ
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > b thêm --debug  rồi xem bumper nó báo log ra sao lúc reset robot, dns b set như kia thì ok rồi, mình ko dùng hass nên ko rõ, trc giờ toàn dùng app ecovacs thôi
> > 
> > 
> > > 
> > 
> > 
> > > Vâng. Để tối em đi làm về em chạy lệnh debug xem sao. Còn vấn đề hass. Em nghĩ nó chưa kết nối đc với bumper nên khả năng ko add đc. Qua em cũng ha_ecovacs kia rồi 😃 Còn git của bác. Chạy đc trên raspbian chứ ạ. Em thì ko phải dân chuyên. Chỉ là thích vọv vạch thôi ạ
> > 
> > 
> > b tắt verify_ssl chưa, phải tắt mới đc, git của mình thì mới test trên linux x86_64 (debian), còn arm thì chưa
> 
> Tắt ở đâu bác. Em có thêm dòng verify_ssl trong configuration.yaml của hass rồi bác. Git của bác, chạy rồi có cần config ở đt ko ạ

Tắt rồi thì nó phải đc chứ nhỉ. Git của mình nó chạy như proxy server thôi, b chạy xong ở đt nhập proxy vs port vào là đc, riêng android thì phải patch app để tắt ssl pinning nữa
